File: moban_java/src/main/java/org/jdk/project/dto/config/RobotDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class RobotDto {
  private Long id;
  private String robotId;
  private String robotName;
  private String mac;
  private String robotIp;
  private String roomCode;
  private String deptName;
  private Long regionId;
  private String regionName;
  private String robotType;
  private Integer batteryState;
  private String chargingState;
  private String workingState;
  private String standbyState;
  private String positioningState;
  private Integer enable;
  private String oriCoordinate;
  private Boolean adminMode;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/RobotDto.kt
package org.jdk.project.dto.config

data class RobotDto(
    var id: Long? = null,
    var robotId: String? = null,
    var robotName: String? = null,
    var mac: String? = null,
    var robotIp: String? = null,
    var siteId: Long? = null,
    var siteCode: String? = null,
    var siteName: String? = null,
    var pointId: Long? = null,
    var pointName: String? = null,
    var robotType: String? = null,
    var batteryState: Int? = null,
    var chargingState: String? = null,
    var workingState: String? = null,
    var standbyState: String? = null,
    var positioningState: String? = null,
    var enable: Int? = null,
    var oriCoordinate: String? = null,
    var adminMode: Boolean? = null,
    var remark: String? = null,
)
